charts/service-binding-operator/README.md

```diff
@@ -0,0 +1,88 @@
+
+# Service Binding Operator Helm Chart
+
+This helm chart defines the Service Binding Operator. This provides an option for users to install  service binding operators using helm chart. 
+
+The chart installation will result in the creation of three Custom Resource Definitions(CRDs):
+- bindablekinds.binding.operators.coreos.com
+- servicebindings.binding.operators.coreos.com
+- servicebindings.servicebinding.io
+
+The resources required for service binding operator will also be installed. The chart creates a service-binding-operator namespace and installs the required resources in that namespace. 
+
+## Introduction
+
+The following are the values that can be customized when the chart is installed:
+
+- image.pullPolicy
+- keepTestResources
+
+A user can define values for the image  PullPolicy. As part of helm test we clear up the deploymemt,service binding resource and secret created. If a user is interested to view them , then he has to install the chart with keepTestResources set to true.
+
+## Helm Chart Installation 
+
+The helm chart installation involves the following steps:
+- Add the helm chart repository
+- Install the chart
+- Helm test
+
+Note: If service binding operator is not installed through OLM, the operator requires that cert-manager is available on the cluster. It can be installed by:
+
+kubectl apply -f https://github.com/jetstack/cert-manager/releases/download/v1.6.0/cert-manager.yaml
+
+### Add the helm chart repository
+You need to add our helm repository to your local repository. Name the repository as per your convenience.  
+
+```
+helm repo add service-binding-operator https://redhat-developer.github.io/service-binding-operator-helm-chart/
+```
+
+### Install the helm chart
+```
+helm install service-binding-operator https://redhat-developer.github.io/service-binding-operator-helm-chart/service-binding-operator
```

Comment:
there is not need to do this at all, since the repo was added in previous step. I should be calling the install referencing the repo in the chart. I would add a step to call: helm search repo service-binding-operator to list the chart first, then you know how to reference it during the install

Reply:
The line is wrong because the snippet:

```
helm install service-binding-operator https://redhat-developer.github.io/service-binding-operator-helm-chart/service-binding-operator

```

states that the chart is installed from the given url - and that url should point to the chart tarball, not repo.

Reply:
No actually service-binding-operator is the name of our chart. I will rename the chart repo ,chart release so as to give better clarity.In fact since i have added repo with name service-binding-operator i can even install it with:
helm install <release-name> service-binding-operator/service-binding-operator.